### DISCUSSION TITLE
UI modification guideline

### DISCUSSION THREAD

**Discussion by anhp58** [TerriaJS/terriajs#5531] [Q&A] [answered]
Hi TerriaJS supporters,

I would like to adjust the side panel for my personal purpose.
I have successfully installed the TerriaJS development environment.
I have change the  **"terriajs/lib/ReactViews/SidePanel/SidePanel.jsx"** file but the side panel remained unchanged when I reloaded the browser.
Do I need to do any further steps to make it work as I am new to ReactJS?

Thank you very much for your support.
Regards,

Phan Anh

**Comment by hardreddata** [comment, marked as answer]
Perhaps `npm run gulp` to build the new system? I haven't used the developer environment and I am just guessing here.

**Reply by anhp58** [reply]
Thank you @RussellGrew. That solved my problem.
